### PR TITLE
Add /changesetting command

### PR DESCRIPTION
**Pull request by PHGaming121**
Usage: /cs [Option Name] [Value]

changing Impostors Amount or Players Amount starts the game, because this way, we can do a glitch that can do it in official servers (TIP: create a lobby, then change the impostors/players number when u are alone. After starting and rejoining lobby, share your code)
**NOTE: if u leave the game, u will need to set players/impostors amount back to a supported value, if u dont, u will always get kicked when trying to create a lobby**

Credits: MoreGamemodes

### CODE REVIEW

**Comment by PHGaming121** [comment]
bro, why my old commits are here 💀

File: Patches/MainMenuManagerPatch.cs [hunk 1]
```diff
@@ -50,6 +50,8 @@ public static void SetOnline_Postfix()
         LateTask.New(() => { IsOnline = true; }, 0.1f, "Set Online Status");
     }
 
+    [HarmonyPatch(typeof(SignInStatusComponent), nameof(SignInStatusComponent.SetOnline)), HarmonyPostfix]
+    public static void SetOnlinePostfix() {LateTask.New(() => { IsOnline = true; }, 0.1f, "Set Online Status"); }
```
Comment (Gurge44) [inline Patches/MainMenuManagerPatch.cs:54]: lol this is already there above it

**Comment by PHGaming121** [comment]
just like this?

**Comment by Gurge44** [comment]
yes

**Pull request merged**